build.rs

```diff
@@ -60,37 +60,37 @@ fn main() {
                .include("depend/secp256k1/src")
                .debug(true)
                .flag_if_supported("-Wno-unused-function") // some ecmult stuff is defined but not used upstream
-               .define("SECP256K1_BUILD", Some("1"))
+               .define("SECP256K1_BUILD", "1")
                // TODO these three should be changed to use libgmp, at least until secp PR 290 is merged
-               .define("ENABLE_MODULE_ECDH", Some("1"))
-               .define("USE_EXTERNAL_DEFAULT_CALLBACKS", Some("1"))
+               .define("ENABLE_MODULE_ECDH", "1")
+               .define("USE_EXTERNAL_DEFAULT_CALLBACKS", "1")
                .define("ECMULT_WINDOW_SIZE", Some("15")); // This is the default in the configure file (`auto`)
```

Comment:
I don't know why you changed this but then this line should be changed too.

Reply:
Opss I used replace to replace `Some("1")` with `"1"`, because the Some isn't needed and it just clatters the important parts(IMHO), i'll amend that too